src/tests/ftest/ior/ior_intercept_basic.py

```diff
@@ -47,13 +47,26 @@ def test_ior_intercept(self):
         :avocado: tags=iorinterceptbasic
         """
         apis = self.params.get("ior_api", '/run/ior/iorflags/ssf/*')
+        match_results = []
+        patterns = "^\[libioil\] Performed [0-9]+ reads and [0-9]+ " \
```

Comment:
(lint) '\['. String constant might be missing an r prefix.
(lint) '\]'. String constant might be missing an r prefix.